lib/components/app/app-menu.tsx

```diff
@@ -317,12 +401,16 @@ const IconAndLabel = ({
             {
               id: 'components.AppMenu.menuItemIconAlt'
             },
-            { label }
+            { label: typeof label === 'string' ? label : '' }
```

Comment:
a11y: The icons are purely decorative here, so `alt` text can be set to an empty string (see https://www.w3.org/WAI/tutorials/images/decorative/). Delete the corresponding entry from the language YML files.